pages/doc/integrations_custom_app_metrics_howto.md

```diff
@@ -55,6 +55,8 @@ Spring Boot is commonly used to build apps running on TAS.
 
 Spring Boot Actuator supports both push and pull models, with the only difference typically being configuration changes. In that case, it becomes relatively easy to switch from push to pull or vice-versa.
 
+See also the VMware Tanzu solutions workbook [Instrumenting TAS OpenTelemetry for Spring Boot Application](https://docs.vmware.com/en/VMware-Tanzu-Reference-Architecture/services/tanzu-solutions-workbooks/solution-workbooks-TAS-OpenTelemetry-SpringBoot-TO.html).
```

Comment:
Q: Do all TAS customers get Wavefront access? or do users already need to have a Wavefront account (I think this is the case but it's better to check).

If you are using Tanzu Application Service (TAS) and if you are an existing Aria Operations for Applications customer, you can use the VMware Tanzu solutions workbook. For more information, see [Instrumenting TAS OpenTelemetry for Spring Boot Application](https://docs.vmware.com/en/VMware-Tanzu-Reference-Architecture/services/tanzu-solutions-workbooks/solution-workbooks-TAS-OpenTelemetry-SpringBoot-TO.html).

Reply:
This page is for TAS customers (it's for the TAS integration), so no need add the "if" clause, I think.

Reply:
"To learn more" sounds as if the previous paraph is covered in the workbook but it's not. Changed to:
"To use a VMware Tanzu solutions workbook, see..." What do you think?

Reply:
sounds great!

Reply:
Thank you! Could you please review?